### PR TITLE
milestone 1 changes

### PR DESCRIPTION
**Pull request by afloresOSI**
Output is here: https://osisoft-dev.zoominsoftware.io/bundle/ocs-ictoria-milestone1/page/add-organize-data/collect-data/software-mgmt/software-management.html

### CODE REVIEW

File: content/add-organize-data/collect-data/software-mgmt/configure-edge-system.md [hunk 1]
```diff
@@ -8,17 +8,27 @@ You can import an edge system configuration from the field to its corresponding
...
  - PI Adapter for OPC UA
 
-**Note:** An edge system appears on the `Software Management Edge Systems` page once it is writing health data to OCS. For information on configuring health endpoints, see the specific edge system product documentation.
+ - PI Adapter for RDBMS
+
+ - PI Adapter for Structured Data Files
+
+The `namespaceId` in the data and health endpoint URLs defaults to the namespace where the base configuration is created. For example, if the namespace of the base configuration is MyData, the endpoint URL would be `https://website.com/api/v1/Tenants/{tenantId}/Namespaces/MyData/Omf`.
+
+An edge system appears on the `Software Management Edge Systems` page once it is writing health data to OCS. For information on configuring health endpoints, see the specific edge system product documentation.
```
Comment (osisoft-mbishop) [inline content/add-organize-data/collect-data/software-mgmt/configure-edge-system.md:31]:
```suggestion
An edge system appears on the `Software Management Edge Systems` page once it is writing health data to OCS. For information on configuring health endpoints, see the specific edge system product documentation at [docs.osisoft.com](https://docs.osisoft.com/).
```

File: content/add-organize-data/collect-data/software-mgmt/configure-edge-system.md [hunk 2]
```diff
@@ -54,11 +78,11 @@ To edit an edge system configuration and export it for use:
  
    - To select a JSON file that contains the configuration, select **Start With Default**. 
 
-1. **Optional:** In the **Section Select** dropdown list, select the section of the configuration to modify. The default option of `JSON Configuration` shows the entire configuration.
+1. (Optional) In the **Section Select** dropdown list, select the section of the configuration to modify. The default option of `JSON Configuration` shows the entire configuration.
 
 1. Modify the JSON as needed. For configuration guidelines, refer to the specific edge system documentation. 
 
-   Do not include secrets in the configuration. Secrets cannot be stored or exported in a configuration.
+   **WARNING:** Do not include secrets in the configuration. Secrets cannot be stored or exported in a configuration. Client secrets and passwords must be applied directly on the device..
```
Comment (jasnames) [inline content/add-organize-data/collect-data/software-mgmt/configure-edge-system.md:85]:
```suggestion
   **WARNING:** Do not include secrets in the configuration. Secrets cannot be stored or exported in a configuration. Client secrets and passwords must be applied directly on the device.
```

File: content/add-organize-data/collect-data/software-mgmt/base-configurations.md [hunk 3]
```diff
@@ -17,6 +17,12 @@ You can create and edit base configurations for edge systems in OCS. Once the co
  - PI Adapter for Modbus TCP
  
  - PI Adapter for OPC UA
+ 
+ - PI Adapter for MQTT
+ 
+The `namespaceId` in the data and health endpoint URLs defaults to the namespace where the base configuration is created. For example, if the namespace of the base configuration is MyData, the endpoint URL would be `https://website.com/api/v1/Tenants/{tenantId}/Namespaces/MyData/Omf`.
+
+Use mustache tokens, in the format `{{SecretA}}`, to denote secrets in configuration files. Secrets are managed using clients. The token name must match an Edge Management System property defined for the edge system. Tokens must be used in fields where `isEncrypted=true`. After deploying the configuration to the device, you must manually update configuration on the device with the secret.
```
Comment (jasnames) [inline content/add-organize-data/collect-data/software-mgmt/base-configurations.md:25]:
```suggestion
Use mustache tokens, in the format `{{SecretA}}`, to denote secrets in configuration files. Secrets are managed using clients. The token name must match an Edge Management System property defined for the edge system. Tokens must be used in fields where `isEncrypted=true`. After deploying the configuration to the device, you must manually update the configuration on the device with the secret.
```

**Review by jasnames — CHANGES_REQUESTED** [review]

File: content/add-organize-data/collect-data/software-mgmt/base-configurations.md [hunk 4]
```diff
@@ -36,11 +42,11 @@ To create a edge system base configuration and export it for use:
 
 1. To import a configuration, select **Import Configuration**, then browse to the JSON file that contains the configuration, and select **Import**.
 
-1. **Optional:** In the **Section Select** dropdown list, select the section of the configuration to modify. The default option of `JSON Configuration` shows the entire configuration.
+1. (Optional) In the **Section Select** dropdown list, select the section of the configuration to modify. The default option of `JSON Configuration` shows the entire configuration.
 
 1. Modify the JSON as needed. For configuration guidelines, refer to the specific edge system documentation.
 
-   Do not include secrets in the configuration. Secrets cannot be stored or exported in a configuration.
+   **WARNING:** Do not include secrets in the configuration. Secrets cannot be stored or exported in a configuration. Client secrets and passwords must be applied directly on the device.
```
Comment (osisoft-mbishop) [inline content/add-organize-data/collect-data/software-mgmt/base-configurations.md:49]: Might be helpful to add a reason. Your call though.

```suggestion
   **WARNING:** For security reasons, do not include secrets in the configuration. Secrets cannot be stored or exported in a configuration. Client secrets and passwords must be applied directly on the device.
```

File: content/add-organize-data/collect-data/software-mgmt/base-configurations.md [hunk 5]
```diff
@@ -74,9 +80,9 @@ To modify a base configuration and export it for use:
 
 1. Modify the JSON as needed. For configuration guidelines, refer to the specific edge system documentation.
 
-   Do not include secrets in the configuration. Secrets cannot be stored or exported in a configuration.
+  **WARNING:** Do not include secrets in the configuration. Secrets cannot be stored or exported in a configuration. Client secrets and passwords must be applied directly on the device.
```
Comment (osisoft-mbishop) [inline content/add-organize-data/collect-data/software-mgmt/base-configurations.md:83]: Same comment

```suggestion
  **WARNING:** For security reasons, do not include secrets in the configuration. Secrets cannot be stored or exported in a configuration. Client secrets and passwords must be applied directly on the device.
```

**Review by osisoft-mbishop — APPROVED** [review]
You saw my comments and addressed. Looks good otherwise.

**Review by jasnames — APPROVED** [review]

**Pull request merged**